src/fiber/channel.rs

```diff
@@ -926,57 +916,34 @@ where
 
-        if command.force {
-            if let Some(transaction) = &state.latest_commitment_transaction {
-                self.network
-                    .send_message(NetworkActorMessage::new_event(
-                        NetworkActorEvent::CommitmentTransactionPending(
-                            transaction.clone(),
-                            state.get_id(),
-                        ),
-                    ))
-                    .expect(ASSUME_NETWORK_ACTOR_ALIVE);
-
-                state.update_state(ChannelState::ShuttingDown(
-                    ShuttingDownFlags::WAITING_COMMITMENT_CONFIRMATION,
-                ));
-            } else {
-                return Err(ProcessingChannelError::InvalidState(
-                    "Force shutdown without a valid commitment transaction".to_string(),
-                ));
-            }
-        } else {
-            self.network
-                .send_message(NetworkActorMessage::new_command(
-                    NetworkActorCommand::SendFiberMessage(FiberMessageWithPeerId::new(
-                        self.get_remote_peer_id(),
-                        FiberMessage::shutdown(Shutdown {
-                            channel_id: state.get_id(),
-                            close_script: command.close_script.clone(),
-                            fee_rate: command.fee_rate,
-                            force: command.force,
-                        }),
-                    )),
-                ))
-                .expect(ASSUME_NETWORK_ACTOR_ALIVE);
+        self.network
+            .send_message(NetworkActorMessage::new_command(
+                NetworkActorCommand::SendFiberMessage(FiberMessageWithPeerId::new(
+                    self.get_remote_peer_id(),
+                    FiberMessage::shutdown(Shutdown {
+                        channel_id: state.get_id(),
+                        close_script: command.close_script.clone(),
+                        fee_rate: command.fee_rate,
+                    }),
+                )),
+            ))
+            .expect(ASSUME_NETWORK_ACTOR_ALIVE);
 
-            let shutdown_info = ShutdownInfo {
-                close_script: command.close_script,
-                fee_rate: command.fee_rate.as_u64(),
-                signature: None,
-            };
-            state.local_shutdown_info = Some(shutdown_info);
-            state.update_state(ChannelState::ShuttingDown(
-                flags | ShuttingDownFlags::OUR_SHUTDOWN_SENT,
-            ));
-            debug!(
-                "Channel state updated to {:?} after processing shutdown command",
-                &state.state
-            );
+        let shutdown_info = ShutdownInfo {
+            close_script: command.close_script,
+            fee_rate: command.fee_rate.as_u64(),
+            signature: None,
+        };
+        state.local_shutdown_info = Some(shutdown_info);
+        state.update_state(ChannelState::ShuttingDown(
```

Comment:
suppose the scenario node2 is offline, then node1 shotdown force, what will happen if `node2` restarted the node, `node2` will try to reestablish or maybe should also update the channel status too. maybe we need to add some code for node2 to handle this scneario.